larray/io/session.py

```diff
@@ -164,7 +169,7 @@ def list(self):
 
     def _read_array(self, key, *args, **kwargs):
         df = self.handle.parse(key, *args, **kwargs)
-        return df_aslarray(df)
+        return df_aslarray(df, raw=True)
```

Comment:
What's the relation with the rest? Is this an extra fix? If so it needs a test & changelog :)